### ISSUE TITLE
Incorporate numba with scipy.special and numba-stats

### ISSUE THREAD

**Issue by deanm0000** [erdogant/distfit#42]
There may be some potentially significant speed improvements by running code that is compiled. At first glance, it seems that there doesn't exist a fit method in [numba-stats](https://pypi.org/project/numba-stats/) so that might be a significant hurdle and not all the distributions are available that way so this is kind of a place holder or an aspirational enhancement.

**Comment by erdogant** [comment]
Interesting. I have been looking into it but the number of available distributions is limited.
In this line, I again tried to use `numba` but it keeps giving the same error since forever: https://github.com/numba/numba/issues/3250

Then I looked into `numba-scipy` but I could not find the documentation about distributions. Maybe it is not supported.

**Comment by deanm0000** [comment]
numba-scipy opens up the [scipy.special](https://docs.scipy.org/doc/scipy/reference/special.html) namespace so that numba is aware of them.

**Comment by erdogant** [comment]
I just created a new release where I added parallelization to the bootstrap function which significantly speeds up the results. However, when using multiple cores, it randomly also results in a _RuntimeWarning: invalid value encountered_. I am not sure why this happens but  if you set it back to  `n_jobs=1`, it does not show any _RuntimeWarnings.

Regarding, scipy-special, I am not sure how these functions match with scipy.stats?
